src/SwedbankPay.Sdk/PaymentInstruments/Invoice/InvoicePaymentRequest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SwedbankPay.Sdk.Common;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    public class InvoicePaymentRequest
+    {
+        public InvoicePaymentRequest(Operation operation,
+                              PaymentIntent intent,
+                              CurrencyCode currency,
+                              List<IPrice> prices,
+                              string description,
+                              string userAgent,
+                              CultureInfo language,
+                              IUrls urls,
+                              PayeeInfo payeeInfo,
+                              InvoiceType invoiceType,
+                              bool generatePaymentToken = false,
+                              bool generateReccurenceToken = false,
+                              string payerReference = null,
+                              Dictionary<string, object> metadata = null,
+                              string paymentToken = null,

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)